apps/coscrad-frontend/src/emotion-components/index.ts

```diff
@@ -0,0 +1,9 @@
+import styled from '@emotion/styled';
+import { Card } from '@mui/material';
+import { primaryColor as primary } from './colors';
+
+export const PrimaryCard = styled(Card)`
```

Comment:
Clever use of tagged template literals by whoever wrote this lib. Star the repo, haha.